### PR TITLE
Null Safety

### PR DESCRIPTION
**Pull request by fareesh**
Null Safety

### CODE REVIEW

**Comment by iDeBugger** [comment]
@TechnoUrmish Could you take a look please?

**Review by joshbenaron — APPROVED** [review]

**Review by dmorawetz — APPROVED** [review]

**Comment by calvintam236** [comment]
@fareesh I think you need to fix conflicts first

File: pubspec.yaml [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 description: A flutter plugin for Easily make Flutter apps responsive. Automatically adapt UI to different screen sizes. Responsiveness made simple.
-version: 1.1.7
+version: 2.0.0
 #author: Urmish Patel <urmishpatel9@gmail.com>
 homepage: https://github.com/TechnoUrmish/Sizer
 
 environment:
-  sdk: ">=2.7.0 <3.0.0"
-  flutter: ">=1.17.0 <2.0.0"
+  sdk: ">=2.12.0-0 <3.0.0"
```
Comment (calvintam236) [inline pubspec.yaml:8]: @fareesh I think this should not be pre-release version? This needs to be updated.

File: pubspec.yaml [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+  flutter: ">=2.0.0"
 
 dependencies:
   flutter:
```
Comment (calvintam236) [inline pubspec.yaml:12]: @fareesh these two lines are not needed here when the constraints is set in `environments`.

**Pull request closed without merging**